packages/forma-36-react-components/src/components/Card/Card.tsx

```diff
@@ -27,13 +23,17 @@ export interface BaseCardProps {
    * Class names to be appended to the className prop of the component
    */
   className?: string;
+  /**
+   * Child nodes to be rendered in the component
+   */
+  children?: React.ReactNode;
```

Comment:
Nit: import `ReactNode` from react like MouseEvent, etc.

Reply:
Ah ok! I will change it
thanks!